app/helpers/money_helper.rb

```diff
@@ -1,5 +1,6 @@
 module MoneyHelper
-  def value_with_currency_unit(value, currency)
+  def value_with_currency_unit(value, currency = nil)
+    currency = 'gbp' if currency.nil?
```

Comment:
It might be worth look at [this article](https://www.grzegorowski.com/rails-number_to_currency-change-locale-defaults/) that describes a more standard way of defining `number_to_currency` behaviour.